NSS-Day-Cohort-66/rare-api-dapper-pirates
# Cts post details #43

I added a `PostAuthorSerializer` with an `author` field to the `posts.py` module in the `views` directory.  Then, I used that serializer in the `PostSerializer` to a `user` field.

### To Test:

1. From your `main` branch in your terminal, run `git fetch` then `git checkout cts-post-details` branch.
2. Run `pipenv shell` and then `pipenv install`.
3. Run `code .` to open the project in VSCode.
4. Start the debugger (if you asked which debugger to use, select Django).
5. Open Postman, and run a new `Get` request to `http://localhost:8000/posts/1` (make sure you have `Authorization` included in your headers with `token <your authorization token here>` as its value.
6. Confirm that the response is a javascript object that includes `user` as one of its properties with an object as a value.  This object should contain only a key called "author" with the name of the author as its "value."
